flake.nix

```
@@ -42,7 +45,10 @@
         };
 
         nixosModules = {
-          nvf = import ./flake/modules/nixos.nix self.packages lib;
+          nvf = import ./flake/modules/nixos.nix {
+            inherit lib inputs;
+            inherit (self) packages;
+          };
```

```suggestion
          nvf = import ./flake/modules/nixos.nix {
            inherit lib self;
          };
```

`inputs` and `packages` are both in `self`. Passing packages here makes it more confusing.